src/app/ui/ui.service.ts

```diff
@@ -0,0 +1,24 @@
+import { DOCUMENT } from '@angular/common'
+import { Inject, Injectable } from '@angular/core'
+import { BehaviorSubject } from 'rxjs'
+
+@Injectable({ providedIn: 'root' })
+export class UiService {
```

Comment:
Thoughts on what we talked yesterday about making this a "Global" Component Store?

Reply:
Yep, seems like a good one! Feel free to give it a shot, or we can do this live when we start out?